src/metabase/query_processor/util/add_alias_info.clj

```diff
@@ -257,10 +258,22 @@
   (when-let [[_ _ {::keys [desired-alias]}] (matching-field-in-source-query inner-query field-clause)]
     desired-alias))
 
+(defmulti ^String field-reference
+  "Generate a reference for the field instance `field-inst` appropriate for the driver `driver`.
+  By default this is just the name of the field, but it can be more complicated, e.g., take
+  parent fields into account."
+  {:added "0.46.0", :arglists '([driver field-inst])}
+  driver/dispatch-on-initialized-driver
+  :hierarchy #'driver/hierarchy)
```

Comment:
You should mention this new method in the driver changelog in `docs/developers-guide` so third-party driver authors are aware of it

Reply:
Thanks for the reminder! 👍